<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraForm</class>
 <widget class="QWidget" name="CameraForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="toolButtonScale">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../Lib/Ui/Ui.qrc">
            <normaloff>:/Icons/Resize.png</normaloff>:/Icons/Resize.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frameControls">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>8</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QToolButton" name="toolButtonCancel">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="VideoUi.qrc">
               <normaloff>:/Icons/Cancel.png</normaloff>:/Icons/Cancel.png</iconset>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonSave">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../Player/Player.qrc">
               <normaloff>:/Icons/Save.png</normaloff>:/Icons/Save.png</iconset>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonPlay">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="VideoUi.qrc">
            <normaloff>:/Icons/Play.png</normaloff>:/Icons/Play.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonStop">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="VideoUi.qrc">
            <normaloff>:/Icons/Stop.png</normaloff>:/Icons/Stop.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Состояние</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditCamerStatus">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: lightgray;
color: blue;</string>
          </property>
          <property name="text">
           <string>Соединение</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonReload">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../Lib/DbUi/DbUi.qrc">
            <normaloff>:/Icons/Refresh.png</normaloff>:/Icons/Refresh.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollAreaContent">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>763</width>
           <height>515</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="stackedWidgetCameraPreview">
            <widget class="QWidget" name="pageImage">
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="ImageWithPoints" name="labelCameraPreview">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="../../Lib/Monitoring/Monitoring.qrc">:/Icons/No Video.png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="pageFrame">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="FrameLabel" name="frameLabelCameraPreview" native="true"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../../Lib/DbUi/DbUi.qrc">
     <normaloff>:/Icons/Refresh.png</normaloff>:/Icons/Refresh.png</iconset>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
   <property name="toolTip">
    <string>Обновить изображение</string>
   </property>
  </action>
  <action name="actionScale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Lib/Ui/Ui.qrc">
     <normaloff>:/Icons/Resize.png</normaloff>:/Icons/Resize.png</iconset>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
   <property name="toolTip">
    <string>Автоматическое масштабирование</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../Player/Player.qrc">
     <normaloff>:/Icons/Save.png</normaloff>:/Icons/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Сохранить изменения</string>
   </property>
  </action>
  <action name="actionCancel">
   <property name="icon">
    <iconset resource="VideoUi.qrc">
     <normaloff>:/Icons/Cancel.png</normaloff>:/Icons/Cancel.png</iconset>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="toolTip">
    <string>Отменить изменения</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="VideoUi.qrc">
     <normaloff>:/Icons/Play.png</normaloff>:/Icons/Play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Запустить просмотр камеры</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="VideoUi.qrc">
     <normaloff>:/Icons/Stop.png</normaloff>:/Icons/Stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Остановить просмотр камеры</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageWithPoints</class>
   <extends>QLabel</extends>
   <header>ImageWithPoints.h</header>
  </customwidget>
  <customwidget>
   <class>FrameLabel</class>
   <extends>QWidget</extends>
   <header>FrameLabel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Lib/DbUi/DbUi.qrc"/>
  <include location="../../Lib/Monitoring/Monitoring.qrc"/>
  <include location="../../Lib/Ui/Ui.qrc"/>
  <include location="../Player/Player.qrc"/>
  <include location="VideoUi.qrc"/>
 </resources>
 <connections/>
</ui>
